github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -508,11 +429,11 @@ presubmits:
       - command:
         - /usr/local/bin/runner.sh
         - /bin/bash
-        - -c
+        - -ce
```

Comment:
Note:

This is a bug with github, I didn't change it, it links between the deleted job (had `-c` and the updated job (has and still have `-ce`)

Both are fine because there is only one command (`-ce` is better because it is robust to commands addding and this what we have)